e2e-tests/micronaut-rest-api/src/main/java/com/foo/micronaut/rest/ExperimentalException.java

```diff
@@ -0,0 +1,6 @@
+package com.foo.micronaut.rest;
+
+// runtime exception added to handle the crashing scenario
```

Comment:
comment on classes should be inside `/** */` (note the opening double * ), as that generate JavaDocs that are handled by the IDE